### PR TITLE
US10380 - Reskin /search

### PR DESCRIPTION
**Pull request by seancdavis**
Updated markup for the search page.

This also bumps the version of crds-styles to take advantage of the newer search styles.

---

This is now corresponds to crdschurch/crds-styles#247.

### CODE REVIEW

File: crossroads.net/app/search/search-results.html [hunk 1]
```diff
@@ -1,29 +1,27 @@
-<div class="row">
+<div class="row search-results">
```
Comment (zarahzachz) [inline crossroads.net/app/search/search-results.html]: Is the `search-results` class doing anything?

File: crossroads.net/app/search/search-results.html [hunk 2]
```diff
@@ -1,29 +1,27 @@
...
+            </svg>
+          </button>
+        </span>
+      </div>
+    </form>
     <!-- /search box -->
-    <ul class="local-nav" ng-if="search.showResults">
-      <li ng-class="{active:!search.type}"><a ui-sref="search({type:'',tag:''})">All</a>
+
+    <ul class="local-nav list-inline font-size-smaller" ng-if="search.showResults">
```
Comment (zarahzachz) [inline crossroads.net/app/search/search-results.html]: Does `local-nav` do anything? I tried removing it in dev tools and didn't see any noticeable effects.

File: crossroads.net/styles/main.scss [hunk 3]
```diff
@@ -1,7 +1,15 @@
 .crds-styles {
   @import '~crds-styles/assets/stylesheets/bootstrap';
 
-
+  // Searchbar
+  .searchbar input.form-control.ng-touched {
+    &.ng-valid,
+    &.ng-invalid {
+      border-color: lighten(rgba(231,231,231,1), 5);
```
Comment (zarahzachz) [inline crossroads.net/styles/main.scss]: Is there a reason we're not pulling in a variable for this color value?

File: crossroads.net/app/search/search-results.html (hunk 1 above)
Comment (seancdavis) [reply crossroads.net/app/search/search-results.html]: Nope.

File: crossroads.net/app/search/search-results.html (hunk 2 above)
Comment (seancdavis) [reply crossroads.net/app/search/search-results.html]: It was with legacy styles, but no more!

File: crossroads.net/styles/main.scss (hunk 3 above)
Comment (seancdavis) [reply crossroads.net/styles/main.scss]: Good catch. This is available as a variable, so I shall use that.

**Comment by galiotoweb** [comment]
Looks great. Just one small change:

The <h1> is supposed to be color:#4d4d4d or $cr-gray-dark per Rob's design.

File: crossroads.net/app/search/search-results.html [hunk 4]
```diff
@@ -1,29 +1,27 @@
 <div class="row">
   <div class="col-md-8 col-md-offset-2 col-xs-12">
+    <h1 class="font-family-base-light">search</h1>
```
Comment (galiotoweb) [inline crossroads.net/app/search/search-results.html]: Looks great. Just one small change:

The h1 is supposed to be color:#4d4d4d or $cr-gray-dark per Rob's design.

Comment (galiotoweb) [inline crossroads.net/app/search/search-results.html]: Looks great. Just one small change:

The h1 is supposed to be color:#4d4d4d or $cr-gray-dark per Rob's design.

**Comment by galiotoweb** [comment]
When I checked this in maestro, the search bar is breaking:
screen shot 2017-11-07 at 1 34 41 pm
screen shot 2017-11-07 at 1 34 51 pm

**Pull request merged**